sqlmodel/sql/_relationship_types.py

```diff
@@ -0,0 +1,3 @@
+from typing_extensions import Literal
+
+OnDeleteType = Literal["CASCADE", "SET NULL", "RESTRICT", "NO ACTION"]
```

Comment:
Nice these literals!

As this is used in a single place, I'm thinking that maybe this type could be created there in the same `main.py`. As it's not imported anywhere else, and we don't have any other types to put here, at least yet.

Also to keep the directory structure as similar to SQLAlchemy's as possible.

Reply:
Done